Sofa/framework/Core/src/sofa/core/topology/Topology.h

```diff
@@ -135,8 +135,6 @@ SOFA_CORE_API extern const unsigned int trianglesOrientationInTetrahedronArray[4
 /// List of pair of vertex indices (edge) in a hexahedron
 SOFA_CORE_API extern const unsigned int edgesInHexahedronArray[12][2];
 /// List of 4 vertex indices (quad) in a hexahedron
-SOFA_CORE_API extern const unsigned int quadsInHexahedronArray[6][4];
```

Comment:
Instead of directly removing this constant, it should be first declared as a `DeprecatedAndRemoved` to warn the potential user(s).